packages/editor/src/components/block-list-appender/style.scss

```diff
@@ -22,3 +22,14 @@
 		font-weight: bold;
 	}
 }
+
+// Cater for dark themes
```

Comment:
I would simplify this comment a bit, to match existing instances:

https://github.com/WordPress/gutenberg/search?q=is-dark-theme&unscoped_q=is-dark-theme

I know those are all over the place, but this one seems best:

`// Use opacity to work in various editor styles.`

Reply:
Done